### DISCUSSION TITLE
What are the advantages of using eidos over something like Obsidian?

### DISCUSSION THREAD

**Discussion by cvyl** [mayneyao/eidos#157] [Q&A] [answered]
As the title states. I am curious to learn more about this program but don't know if it has any advantages over using it over something like Obsidian.md.

**Comment by mayneyao** [comment, marked as answer]
Eidos largely draws inspiration from Obsidian. Rather than calling it an offline Notion, I prefer to compare eidos to Obsidian.sqlite. I really like Notion's Database feature, which Obsidian lacks. Although plugins like dataview can compensate for this experience to some extent, data queries based on markdown still have too many limitations.

Eidos is possibly the best-performing airtable-like database that's available offline without installation, capable of handling hundreds of thousands to millions of records in a single table. If you have structured data management needs, eidos might be more suitable. Eidos defines itself as an extension framework serving PDM (Personal Data Management), rather than just PKM (Personal Knowledge Management), and it's not merely a note-taking app.

**Reply by cvyl** [reply]
This answered my question, thank you very much for your explanation!
